src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -1530,6 +1540,8 @@ enum AppStrings {
 		enum Info {
 			static let title = NSLocalizedString("Statistics_Info_Title", comment: "")
 			static let subtitle = NSLocalizedString("Statistics_Info_Subtitle", comment: "")
+			static let pandemicRadarTitle = NSLocalizedString("Statistics_Info_PandemicRadar_Title", comment: "")
+			static let pandemicRadarText = NSLocalizedString("Statistics_Info_PandemicRadar_Text", comment: "")
```

Comment:
```suggestion
			static let pandemicRadarDescription = NSLocalizedString("Statistics_Info_PandemicRadar_Text", comment: "")
```

Reply:
"Text" is equal to the former ones. Would keep it equally